source/languages/en/global_nav.yml

```diff
@@ -92,6 +92,7 @@ riak:
           - "[[Commit Hooks|Advanced Commit Hooks]]"
           - "[[Replication Properties]]"
           - "[[Strong Consistency|Using Strong Consistency]]"
+          - "[[Write Once Buckets]]"
```

Comment:
My completely unwanted $.02 here is to make the official term "write-once buckets" and, correspondingly, to title the doc "Write-once Buckets" (in accordance with Basho's capitalization/hyphenation standards).

Aaaaaand I'm out.